docs/using/recipes.md

```diff
@@ -581,3 +581,32 @@ FROM jupyter/scipy-notebook:85f615d5cafa
 RUN npm install -g ijavascript
 RUN ijsinstall
 ```
+
+## Add Microsoft SQL Server ODBC driver
+
+The following recipe demonstrates how to add functionality to read from and write to an instance of Microsoft SQL server in your notebook.
+
+```dockerfile
+ARG BASE_CONTAINER=jupyter/datascience-notebook:latest
+FROM $BASE_CONTAINER
+
+USER root
+
+ENV MSSQL_DRIVER "ODBC Driver 18 for SQL Server"
```

Comment:
Is this needed? It looks more like helpful information rather than env variable.

Reply:
The rationale is that you'll need to reference the driver in your connection string, and by defining it in an environment variable, you can seamlessly switch driver on the container without breaking your notebook.

For [example](https://github.com/Realiserad/jupyter-docker-mssql/blob/main/example/notebooks/Test.ipynb):
```
import pyodbc
import os

conn_string = 'DRIVER={'  + os.environ['MSSQL_DRIVER'] + '};SERVER=tcp:db.foo.org;DATABASE=FooDB;UID=k8s;PWD=foo123'
connection = pyodbc.connect(conn_string)
```

It also saves people from googling for the driver name when they copy-paste the recipe (at least I had to, because I'm not overly familiar with Microsoft's ODBC driver).

Reply:
Great, thanks 👍